docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -49,11 +51,24 @@ \section{Team Member Roles}
 
 \section{Workflow Plan}
 
-\begin{itemize}
-	\item How will you be using git, including branches, pull request, etc.?
-	\item How will you be managing issues, including template issues, issue
-	classificaiton, etc.?
-\end{itemize}
+We will be using git as a version control software with GitHub as our remote repository. The root branch will be \code{master} and development branch mased on \code{master} will be \code{dev}. The developer will branch off from \code{dev} to work on issues. There will be a new branch for each issue and will be named comtaining the issue number and name detailed below. Continuous Inntegration is set up between all branches \code{dev}, \code{master}, and Issues.
+
+Issues are used to track tasks and meetings related to the project. Meeting issues will be created before each meeting according to the \textbf{MeetingIssueTemplate.md} template. These include addentance, minutes, and action items from the meeting. These notes can be referenced from other issues and PRs for tracability. 
+Feature issues are used to track tasks. They can be referenced in PRs and meetings and outline the tesk that the developer must fullfill in their PR. 
```

Comment:
Feature issues are used to track tasks. They can be referenced in PRs and meetings _to outline the task that the developer must fulfill in their PR_. 

- I think using to instead of that second and makes it flow a bit better + spelling errors